src/core/DialogBackground.qml

```diff
@@ -67,25 +67,24 @@ Rectangle {
                 border.color: "black"
                 border.width: 2
 
-                Image {
-                    id: titleIcon
-                    anchors {
-                        left: parent.left
-                        top: parent.top
-                        margins: 4 * ApplicationInfo.ratio
+                Row {
+                    spacing: 2
+                    padding: 8
+                    Image {
+                        id: titleIcon
                     }
-                }
 
-                GCText {
-                    id: title
-                    text: dialogBackground.title
-                    width: dialogBackground.width - 30
-                    horizontalAlignment: Text.AlignHCenter
-                    verticalAlignment: Text.AlignVCenter
-                    color: "black"
-                    fontSize: 20
-                    font.weight: Font.DemiBold
-                    wrapMode: Text.WordWrap
+                    GCText {
+                        id: title
+                        text: dialogBackground.title
+                        width: dialogBackground.width - (30 + cancel.width)
```

Comment:
There is a small overriding: https://pasteboard.co/H4dTEPf.png

Reply:
Recalculated it. Now it doesn't override in any activity.

Reply:
is the change commited? It's the same for now

Reply:
No, it is not i'm still working on other changes.